### PR TITLE
📖  Update quick-start.md to include installation steps using clusterctl

### PR DESCRIPTION
**Pull request by Jont828**
**What this PR does / why we need it**: Update installation steps in quick start to use `clusterctl init --addon helm`

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

**Comment by Jont828** [comment]
@dtzar I've updated the docs based on what we talked about, PTAL when you get the chance

File: docs/quick-start.md [hunk 1]
```diff
@@ -11,23 +11,32 @@ The prerequisites include:
...
+
+##### Alternative installation steps: 
 
 Run the following command to clone the CAAPH repository into your Go src folder:
 
 ```bash
 $ git clone git@github.com:kubernetes-sigs/cluster-api-addon-provider-helm.git ${GOPATH}/src/cluster-api-addon-provider-helm
 ```
 
-### 4. Install CAAPH to the management cluster
```
Comment (dtzar) [inline docs/quick-start.md:29]: Jumps from 3 to 5 now in overall flow.  Need to renumber below steps.

**Review by dtzar — CHANGES_REQUESTED** [review]
Can you also please provide more surrounding YAML context let's say for a sample AKS cluster with this chunk?
```yaml
nginxIngressChart: enabled
```

**Comment by Jont828** [comment]
@dtzar Done, how does this look?

**Review by dtzar — APPROVED** [review]

**Pull request merged**